zerver/context_processors.py

```diff
@@ -85,6 +85,9 @@ def zulip_default_context(request):
     if settings.ZILENCER_ENABLED:
         apps_page_url = '/apps/'
 
+    path_parts = [chunk for chunk in request.path.split('/') if chunk != '']  # type: List[str]
+    page_is_help_center = 'help' in path_parts
```

Comment:
Ah, great. This makes way more sense. I'll insert commit (right after the first 3) that removes the `path` attribute that was added to `HostRequestMock` and `POSTRequestMock` along with the check on `request.path`.